package.json

```
@@ -14,18 +14,21 @@
   },
   "scripts": {
     "build": "bash ./build.sh",
-    "test": "mocha test/test-*.js",
+    "test": "jest",
```

just on thing i noticed, because running the tests required a build, would it make sense to have the following?
```suggestion
    "test": "npm run build && jest",
```

---

Good point! I thought about that, but I felt like I was taking away the option to decide when to build, or what build to test.
I would definitely add a build-test script 👍